ui/DesignSystem/Component/KeyHint.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
```

Comment:
```suggestion
<script lang="ts">
```



get in comrade, we have types now
![bday](https://user-images.githubusercontent.com/2712962/93537064-5edb4a00-f8ff-11ea-965c-e3fd35a4dfb1.jpg)

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/914/commits/4da650c03cb5846cdd55d435f473cb6dde0749fd